### ISSUE TITLE
Convert Checkboxes to `Types::BOOLEAN` to help reducing the database row size

### ISSUE THREAD

**Issue by grummbeer** [hofff/contao-navigation#35]
A few days ago I run into the "Row Size Too Large" error [1]. Contao has changed all checkboxes to `Types::BOOLEAN`[2].

contao-navigation also uses some checkboxes. If you think it is a good idea to reduce the row size, I would create a PR for it.

[1] https://mariadb.com/kb/en/troubleshooting-row-size-too-large-errors-with-innodb/
[2] https://github.com/contao/contao/pull/4797

**Comment by dmolineus** [comment]
Yes, it's a good idea. Internally the fields will be created as tinyint fields though. Therefore we have to ensure that strict type handling would not break anything. Be aware that Contao 4.13 and 5 work differently here. Contao 5.3 has a type handling which will create booleans from the doc block types.

**Comment by grummbeer** [comment]
Thanks for the hint, I'm not that familiar with contao yet.
